rotkehlchen/api/v1/schemas.py

```diff
@@ -2869,18 +2874,25 @@ def validate_schema(
             data: dict[str, Any],
             **_kwargs: Any,
     ) -> None:
-        if data['identifier'] == 1 and data['name'] != ETHEREUM_ETHERSCAN_NODE_NAME:
-            raise ValidationError(
-                message="Can't change the etherscan node name",
-                field_name='name',
-            )
-        # verify that if the node is not etherscan the endpoint field has valid information
-        if data['identifier'] != 1 and len(data['endpoint'].strip()) == 0:
+        if len(data['endpoint'].strip()) != 0:
+            # if it is not etherscan (if the endpoint is not empty), skip.
+            # We only apply validation rules to etherscan modification requests.
+            return
+
+        is_etherscan = self.dbhandler.is_etherscan_node(data['identifier'])
```

Comment:
this is a different check. By doing this you could allow to replace the etherscan endpoint. I believe it wouldn't have any effect but I'm not sure of the side effects that it have. before there was an extra check for the identifier